multinet/db.py

```diff
@@ -222,13 +229,38 @@ def get_workspace_metadata(name: str) -> Workspace:
 
     # Find the metadata record for the named workspace. If it's not there,
     # something went very wrong, so bail out.
-    metadata = workspace_mapping(name)
+    metadata = workspace_mapping.__wrapped__(name)
```

Comment:
Remind me what `.__wrapped__()` does here? It just calls the undecorated version of the function?

Why do we need that here? If you include a comment explaining the need, that would be good.

(Hopefully this becomes moot after the ORM stuff lands but still, in the meantime we should cover our donkeys.)

Reply:
Yeah it calls the un-decorated function. The reason I added that is because when calling `get_workspace_metadata`, we want the most recent version of the metadata, and this function itself isn't cached, so we couldn't just call the un-decorated version of `get_workspace_metadata`, as it doesn't exist. With the changes that I said I'd be making in my other comment, this might actually not be necessary anymore.

I'd still be okay with using the un-cached function, as long as it doesn't impact performance. What do you think?

Reply:
Correctness matters more than performance in this case, so do whatever makes the most sense. I'd just include a comment about why it's this way, just to spare confusion or errors in the future.

Reply:
2b31899 I undid the use of the wrapped function, so I didn't think a comment was necessary anymore